### ISSUE TITLE
<CR> does nothing with default configuration

### ISSUE THREAD

**Issue by yesudeep** [AckslD/muren.nvim#17]
Namaste!

Love the idea. However, when I try to issue the replace command by pressing `<CR>`, nothing happens.

My configuration in `init.lua`:

```  
  {
    'AckslD/muren.nvim',
    config = true,
  },
```

Thank you! :)

Cheers,
Yesudeep.

**Comment by yesudeep** [comment]
Also, would it be possible for the up and down arrow keys to work in 
the options pane with the default keybindings?

**Comment by AckslD** [comment]
Hi @yesudeep, in which window are you in when pressing `<CR>`. Could you check what `nmap <CR>` says?

Regarding the second question, I made up and down arrows to scroll the preview, even when you're in the options window. But you can change these keys by eg:
```lua
  {
    'AckslD/muren.nvim',
    opts = {
      keys = {
        scroll_preview_up = '<your-key>',
        scroll_preview_down = '<your-key>',
      },
    },
  },
```

**Comment by hungnguyen1503** [comment]
I have the same issue

**Comment by AckslD** [comment]
@hungnguyen1503 which issue? It would be helpful to get some more information, for example about what I asked above

**Comment by hungnguyen1503** [comment]
Hi, I installed your plugin last 3 weeks, and it worked very well. but after I sync the latest commit, I can't replace anything by pressing `<CR>`.
Here is my config 
![image](https://user-images.githubusercontent.com/58872428/236976140-9b24a641-8f4c-4e53-8d32-82ab392037f9.png)
Could you please help me to check it? Thank you.

**Comment by AckslD** [comment]
> in which window are you in when pressing `<CR>`. Could you check what `nmap <CR>` says?

**Comment by hungnguyen1503** [comment]
Hi, No error message appears after I press `<CR>` and I have checked out to commit 7ea7023cfb67f9094570888bfa8e02c420c43371, the plugin worked very well, but the preview window isn't highlighted.

**Comment by AckslD** [comment]
I'm sorry but it's very hard to help you if you don't answer my questions above. If you don't understand what I'm asking, feel free to ask me to clarify

**Comment by hungnguyen1503** [comment]
Thank you for your support, I'm going to find my bug and report it to you if I see it.

**Comment by algmyr** [comment]
Check that you are running a recent enough vim version. I had the same issue and it seems it's due to my version being an old enough 0.9 git branch version to not have `nvim_exec2`.

Maybe it's worth somehow failing loudly when `nvim_exec2` doesn't exist. I saw no feedback at all that something had gone wrong, but was able to find the issue digging into the code.

**Comment by AckslD** [comment]
Oh I see, thank you @algmyr! Should be fixed by #18, feel free to let me know if it works

**Comment by algmyr** [comment]
I don't have the old version around anymore, but maybe others with in the thread do
